grails-app/controllers/org/broadinstitute/orsp/SearchController.groovy

```diff
@@ -152,6 +152,7 @@ class SearchController implements UserInfo {
                 options.irbsOfRecord) {
             rows = queryService.findIssues(options).collect {
                 Map<String, Object> arguments = IssueUtils.generateArgumentsForRedirect(it.type, it.projectKey, null)
+                Collection<String> accessContacts = getProjectAccessContact(it.extraPropertiesMap, it.reporter)
```

Comment:
Minor suggestion. Since these values go right to the UI, it might be nice to sort them here. Alpha is probably the safest bet. That way you are generally returning the list in the same order each time and the UI can decide what to do differently if it wants to. The way it is now, the list will be random depending on how it loads.